node-gui/src/backend/backend_impl.rs

```diff
@@ -299,7 +301,36 @@ impl Backend {
                 let client = make_cold_wallet_rpc_client(Arc::clone(&self.chain_config));
 
                 let (wallet_rpc, command_handler, best_block, accounts_info, accounts_data) = self
-                    .create_wallet(client, file_path.clone(), &mnemonic, import, wallet_events)
+                    .create_wallet(
+                        client,
+                        file_path.clone(),
+                        wallet_args,
+                        import,
+                        wallet_events,
+                    )
+                    .await?;
+
+                let wallet_data = WalletData {
+                    controller: GuiHotColdController::Cold(wallet_rpc, command_handler),
+                    accounts: accounts_data,
+                    best_block,
+                    updated: false,
+                };
+
+                (wallet_data, accounts_info, best_block)
+            }
+            #[cfg(feature = "trezor")]
+            (WalletType::Trezor, _) => {
+                let client = make_cold_wallet_rpc_client(Arc::clone(&self.chain_config));
```

Comment:
why cold mode? I thought that's the whole point that hardware wallet is both convenient and secure.